azure-ad.sgnl.yaml

```diff
@@ -573,6 +573,54 @@ entities:
       - name: trustType
         externalId: trustType
         type: String
+  Role:
+    # https://learn.microsoft.com/en-us/graph/api/resources/directoryrole?view=graph-rest-1.0
+    description: "Directory Role Entity in Azure AD"
+    displayName: Role
+    externalId: Role
+    pageSize: 999
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: displayName
+        externalId: displayName
+        type: String
+        indexed: true
+      - name: description
+        externalId: description
+        type: String
+      - name: roleTemplateId
+        externalId: roleTemplateId
+        type: String
+      - name: deletedDateTime
+        externalId: deletedDateTime
+        type: DateTime
+  RoleMember:
+    # https://learn.microsoft.com/en-us/graph/api/directoryrole-list-members?view=graph-rest-1.0    
+    description: "Directory Role Member Entity in Azure AD"
```

Comment:
nit: This is the only catalog that has quotes around the description, we should remove this for all entities